src/components/unsigned-sweep.js

```diff
@@ -172,8 +172,31 @@ class UnsignedSweep extends Component {
       }
 
       this.updateKeysFromIDs(baseCoin, recoveryParams);
+      
+      const userKeyPaths = ['/0/0', '/0'];
+      const noInputErrMsg = 'No input to recover - aborting!';      
```

Comment:
So this is the literary error message from the SDK, in the event that no addresses with balances are found. However, we would not know when SDK ever changes this error message at all. Happy to take feedback on the best way we can check for this kind of error.

Reply:
consider adding a type `class ErrorNoInputToRecover extends Error` in the sdk and using `e instanceof ErrorNoInputToRecover` in the check